settings.py

```diff
@@ -388,10 +388,14 @@
 DOMAIN_NOT_ADMIN_REDIRECT_PAGE_NAME = "homepage"
 
 PAGES_NOT_RESTRICTED_FOR_DIMAGI = (
-    '/a/%(domain)s/settings/project/internal_subscription_management/',
-    '/a/%(domain)s/settings/project/internal/info/',
+    '/a/%(domain)s/settings/project/billing/statements/',
```

Comment:
Just putting out thought here can't list be stored as a populated list in feature flag or in database or something and code just takes list from there. So that you dont need to update the code when the list changes.

Reply:
I think it's good that we need to update the code when the list changes, since we want to make it a very intentional step if we other add other pages that dimagi staff can see even when the customer has opted to restrict access